public/js/app/routes/PostRoute.js

```diff
@@ -5,7 +5,7 @@ define(["app/app",
 
   App.PostRoute = Ember.Route.extend(App.CustomErrorRoute, {
     deactivate: function() {
-      this.controllerFor('pub-sub').unsubscribe()
+      App.PubSubService.unsubscribe()
```

Comment:
I wonder won't it be dangerous to use App.PubSubService directly without requirejs dependency at the very top? Most likely it's pretty safe as it's ember initializer, but let's let keep an eye.